Source/Basic Shapes/SvgVisualElement.cs

```diff
@@ -288,7 +288,7 @@ protected internal virtual bool RenderStroke(ISvgRenderer renderer)
                                     }
 
                                     /* divide by stroke width - GDI behaviour that I don't quite understand yet.*/
-                                    pen.DashPattern = this.StrokeDashArray.ConvertAll(u => ((u.ToDeviceValue(renderer, UnitRenderingType.Other, this) <= 0) ? 1 : u.ToDeviceValue(renderer, UnitRenderingType.Other, this)) /
+                                    pen.DashPattern = this.StrokeDashArray.Select(u => ((u.ToDeviceValue(renderer, UnitRenderingType.Other, this) <= 0) ? 1 : u.ToDeviceValue(renderer, UnitRenderingType.Other, this)) /

```

Comment:
Is this related to the fix?

Reply:
Yes. ConvertAll is a method defined in List<T> but not in ObservableCollection, and do the same thing that Select (which works with any IEnumerable as it is a Linq method).

Reply:
Ah ok, thanks!